sdcm/cluster.py

```diff
@@ -915,6 +920,8 @@ def start_task_threads(self):
             self.start_journal_thread()
             self.start_backtrace_thread()
             self.start_db_log_reader_thread()
+        elif 'monitor' in self.name:
+            self.start_monitor_alert_manager()
```

Comment:
`start_alert_manager_thread` - convention is that if something is starting a thread it should have a `thread` suffix as you see in above threads

Reply:
fixed